NPE in TabbedViewPager

Reported for all beta versions of the current series, Android 10-11, amount 30, users 15

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performDestroyActivity (ActivityThread.java:5309)
  at android.app.ActivityThread.handleDestroyActivity (ActivityThread.java:5339)
  at android.app.servertransaction.DestroyActivityItem.execute (DestroyActivityItem.java:44)
  at android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:176)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:97)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2246)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:233)
  at android.app.ActivityThread.main (ActivityThread.java:8010)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:631)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:978)
Caused by: java.lang.NullPointerException: 
  at cgeo.geocaching.activity.TabbedViewPagerActivity.onDestroy (TabbedViewPagerActivity.java:238)
  at cgeo.geocaching.CacheDetailActivity.onDestroy (CacheDetailActivity.java:436)
  at android.app.Activity.performDestroy (Activity.java:8251)
  at android.app.Instrumentation.callActivityOnDestroy (Instrumentation.java:1364)
  at android.app.ActivityThread.performDestroyActivity (ActivityThread.java:5294)
```

--- a/main/src/cgeo/geocaching/activity/TabbedViewPagerActivity.java
+++ b/main/src/cgeo/geocaching/activity/TabbedViewPagerActivity.java
@@ -250,7 +250,9 @@ public abstract class TabbedViewPagerActivity extends AbstractActionBarActivity 
 
     @Override
     protected void onDestroy() {
-        viewPager.unregisterOnPageChangeCallback(pageChangeCallback);
+        if (viewPager != null) {
+            viewPager.unregisterOnPageChangeCallback(pageChangeCallback);
+        }
         super.onDestroy();
     }
 

